### ISSUE TITLE
Error: Cannot find module './checker'

### ISSUE THREAD

**Issue by gcanti** [danvk/typings-checker#23]
Hi,

this is the tool I was looking for, thanks for writing it.

I just installed the current published version (1.1.1) and it raises the following error

```
> typings-checker --project typings-checker/tsconfig.json typings-checker/index.ts

module.js:341
    throw err;
    ^

Error: Cannot find module './checker'
    at Function.Module._resolveFilename (module.js:339:15)
    at Function.Module._load (module.js:290:25)
    at Module.require (module.js:367:17)
    at require (internal/module.js:16:19)
    at Object.<anonymous> (/node_modules/typings-checker/src/index.js:4:17)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Function.Module.runMain (module.js:447:10)
```

Looking in the `node_modules/typings-checker` folder I can see the following files

```
src
  checker.ts
  index.js
  index.ts
```

Something went wrong while publishing I guess?

**Comment by danvk** [comment]
The `dist/` directory was `.gitignored` which meant it was also npm ignored. I added a separate `.npmignore` file and published an update (1.1.2). Give that a try?

**Comment by gcanti** [comment]
Now works, thanks a lot
